src/main/java/seedu/linkedout/model/util/SampleDataUtil.java

```diff
@@ -4,46 +4,46 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import seedu.linkedout.model.AddressBook;
-import seedu.linkedout.model.ReadOnlyAddressBook;
-import seedu.linkedout.model.person.Address;
-import seedu.linkedout.model.person.Email;
-import seedu.linkedout.model.person.Name;
-import seedu.linkedout.model.person.Person;
-import seedu.linkedout.model.person.Phone;
+import seedu.linkedout.model.Linkedout;
+import seedu.linkedout.model.ReadOnlyLinkedout;
+import seedu.linkedout.model.applicant.Address;
+import seedu.linkedout.model.applicant.Applicant;
+import seedu.linkedout.model.applicant.Email;
+import seedu.linkedout.model.applicant.Name;
+import seedu.linkedout.model.applicant.Phone;
 import seedu.linkedout.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
```

Comment:
{@code AddressBook should no longer exist}

This comment contains a direct reference to a file which doesn't exist. Quite a high priority to fix comment-wise.